### ISSUE TITLE
✨  new Applicant Dashboard page

### ISSUE THREAD

**Issue by brindasasi** [GCTC-NTGC/gc-digital-talent#10772]
## ✨ Feature

Assemble the new Applicant Dashboard page.

> [!IMPORTANT]
>  **OUT OF SCOPE**
> - Employee tools widget (and all tools it contains)
> - Card that says: "Are you a government of Canada employee?"
> - Permanently turning this page on

## 🕵️ Details

Create the "Your information" and "Resources" widgets, then pull together all the parts of the new applicant dashboard.

The Applications and Processes widget should have been completed in #10788.

When the application or process cards are clicked, the dialogues created in #12214 should be shown.

From my github ticket:

### Your information
Unfortunately the designs are a little ahead of where we are. They make the assumption that some of the personal information page was moved to the account settings page (like Name and Contact email). Since that work is not done yet, you'll have to use the copy provided here in this ticket instead of the copy in the designs for the **Your information** widget.

**These 4 links should be presented in this widget:**

- **Personal information** links to `/applicant/personal-information`
   - "Name, contact info, employment equity, language proficiency, and work preferences."
   - Shows as complete if all sections of the Personal information page are complete (existing dashboard has logic for this)
- **Career experience** links to `/applicant/career-timeline`
   - "Work, education, volunteering, awards, and more."
   - Shows as complete if at least one experience has been added (this is new logic, as we don't show completeness on the existing dashboard for Career timeline)
- **Skills portfolio** links to `/applicant/skills`
   - "Manage skills, and edit top skills or skills you'd like to learn."
   - Shows as complete if all sections of the Skills portfolio are complete (existing dashboard has logic for this)
- **Account settings** links to `/applicant/settings`
   - "Learn about GCKey and manage notifications."
   - Always shows as complete

### Resources
The resources section currently just show plain links with descriptive text. Please use the text in the designs for this section:
- **Learn about skills** links to `/skills`
- **Contact support** links to `/support`

## 🧑‍🎨 Designer

@JoshBeveridge 

https://www.figma.com/design/oZhQe9ahidHFfaiqgREvxa/Dashboard-(All-users)?node-id=2011-38235&node-type=frame&t=sltU21dpZZlO5BUt-0

## 📸 Screenshot

Basic applicant dashboard
Screenshot slightly out of date for copy - prefer using text in Details section
![Applicant dashboard](https://github.com/user-attachments/assets/1a59edad-3294-459b-8a20-85c6ae2b7974)

## 🙋‍♀️ Proposed Implementation

Start from what we built for Manager Dashboard.

## ✅ Acceptance Criteria

- [ ] Applications and Processes widget in place
- [ ] Application dialogues functional
- [ ] Process dialogues functional
- [ ] Your information widget in place
- [ ] Completeness indicators for Your information widget functional
- [ ] Resources widget in place
- [ ] feature flag, default false to control routing
  - [ ] might have to remove some Nginx routing
  - [ ] when TRUE
    - [ ] `/applicant/dashboard` navigates to this page
    - [ ] `/applicant` redirects to `/applicant/dashboard`
  - [ ] when FALSE (default)
    - [ ] `/applicant/dashboard` redirects to the old dashboard
    - [ ] `/applicant` continues to navigate to the old dashboard
- [ ] ~design and copy review~

## 🛑 Blockers

Issues which must be completed before this one.

```[tasklist]
- [ ] #10788 
- [ ] #12164 
- [ ] #12214
- [ ] #12236
```

**Comment by tristan-orourke** [comment]
> Actually getting "suspendedAt" working with the Hired status is out of scope for this issue (see https://github.com/GCTC-NTGC/gc-digital-talent/issues/12212)

@gobyrne which should we do for now?
1. showing the suspectedAt form in this case
2. let them set it but not do anything with the result

**Comment by gobyrne** [comment]
2 for now

**Comment by esizer** [comment]
> Career experience links to /applicant/career-timeline

If we are going to start using this term for this section we may want to review the site for this term. Currently we have a lot of references to "Career timeline".

> Learn about skills links to /skills

One thing that could be confusing here is that, in the nav, we have skills under "system settings" but here it is under resources. It feels a little strange to have it appear in two different sections of the site 🤔
